README.md

```diff
@@ -6,7 +6,7 @@ How to compile RINGMesh             {#ringmesh_compiling}
 [![Coverage Status](https://coveralls.io/repos/bitbucket/ring_team/ringmeshcoverage/badge.svg?branch=default)](https://coveralls.io/bitbucket/ring_team/ringmeshcoverage?branch=default)
 
 RINGMesh is tested under Linux (64 bits) and Windows (64 bits).
-You will need CMake (version >= 2.8.11). There is no other dependency (everything
+You will need CMake (version >= 3.0). There is no other dependency (everything
```

Comment:
In the CMakeLists.txt it is indicated a version of cmake of 3.1 and not 3.0. Should be minor but it is not consistent.